index.ts

```diff
@@ -29,11 +33,17 @@ declare module 'vue' {
 interface VueI18NextOptions {
     i18next: i18n;
     rerenderOn?: ('languageChanged' | 'loaded' | 'added' | 'removed')[];
+    // Optional custom pattern for matching slot start of the `TranslationComponent.
+    slotStart?: string,
+    // Optional custom pattern for matching slot end of the `TranslationComponent.
+    slotEnd?: string,
 }
 
 export default function install(app: App, {
     i18next,
     rerenderOn = ['languageChanged', 'loaded', 'added', 'removed'],
+    slotStart,
```

Comment:
We could move the defaults from `slotNamePattern` to this place, so the defaults are obvious from the source code as well.